src/Env.ts

```diff
@@ -9,10 +8,10 @@ export const get = <const E extends Record<PropertyKey, unknown>>() =>
 
 type ExcludeEnv<E, S> =
   E extends Get<Record<PropertyKey, unknown>>
-    ? S extends E['R'] ? never
-    : S extends Record<PropertyKey, unknown>
-      ? Get<{ readonly [K in keyof E['R'] as S[K] extends E['R'][K] ? never : K]: E['R'][K] }>
-      : E
+  ? S extends E['R'] ? never
+  : S extends Record<PropertyKey, unknown>
+  ? Get<{ readonly [K in keyof E['R']as S[K] extends E['R'][K] ? never : K]: E['R'][K] }>
+  : E
   : E
```

Comment:
Ugh, but I don't know how to configure ESLint indent rules to improve the situation